infra/lambda_functions/websocket/disconnect/app.js

```diff
@@ -28,14 +35,12 @@ exports.handler = async (event) => {
     await ddb.delete(deleteParamsConnection).promise();
   } catch (e) {
     console.log("Error when deleting connection:", e);
-    return {
-      statusCode: 500,
-      body: "Failed to disconnect: " + JSON.stringify(e),
-    };
   }
+
+  // get the user data
   const openDiagramsScanParams = {
     TableName: OPEN_DIAGRAMS_TABLE_NAME,
-    ProjectionExpression: "connectionId, diagramId",
+    ProjectionExpression: "connectionId, diagramId, isMaster",
```

Comment:
I think you've put this as a primary key in a later PR, right? That way we can just do a get, which will be much faster than a scan.